python/setup.py

```diff
@@ -381,7 +381,7 @@ def build_extensions(self):
     packages=packages,
     cmdclass={'build_ext': BuildExt},
     data_files=data_files,
-    setup_requires=['numpy'],
+    setup_requires=['numpy==1.16.4'],
```

Comment:
Instead of specifying a version, we should specify something like `numpy<1.17.0`
Please test a solution like that.